<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="tbtnList">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>180</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/new/prefix1/resource/gamelist.png</normaloff>:/new/prefix1/resource/gamelist.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnStartgame">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>540</y>
      <width>180</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/new/prefix1/resource/begin.png</normaloff>:/new/prefix1/resource/begin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="tbtnTuichu">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>540</y>
      <width>180</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/new/prefix1/resource/Tuichu.png</normaloff>:/new/prefix1/resource/Tuichu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
